source/conf.py

```diff
@@ -46,17 +46,17 @@
 
 # General information about the project.
 project = u'dojot'
-copyright = u'2017, CPqD'
-author = u'Matheus Magalhaes'
+copyright = u'2018, CPqD'
+author = u'Giovann Curiel dos Santos, Matheus Magalhaes, Rafael Scaraficci'
```

Comment:
@ericbaum should be here, too, shouldn't he?

Reply:
Yes. All bugs must be reported to him.